srcpkgs/qhull/template

```diff
@@ -1,16 +1,21 @@
 # Template file for 'qhull'
 pkgname=qhull
-version=2019.1
+version=2020.2
 revision=1
 build_style=cmake
+make_build_target="all libqhull"
 short_desc="Computes volumes, surface areas and approximations to the convex hull"
 maintainer="Martin Riese <grauehaare@gmx.de>"
-license="BSD"
+license="Qhull"
```

Comment:
You will need to `vlicense` a copy of this license in `post_install`, as per the terms of the license.

Reply:
Done.